source/playgrounds.txt

```diff
@@ -30,12 +30,40 @@ files with the ``.mongodb`` extension as playgrounds.
 
 .. include:: /includes/admonitions/note-playgrounds.rst
 
-|vsce| splits your Playground window to display the results of your 
-Playground in |json| format in the right-side pane labeled ``Playground 
-Results.json``. If you disabled split-view, |vsce| displays the 
-Playground results in |json| format in a new tab to the right of 
-your Playground. You cannot edit the results in the results pane. 
-You can save the results to a file. 
+- If you enabled split-view, |vsce| splits your Playground window to
+  display the results of your Playground. |vsce| labels this right-side
+  pane :guilabel:`Playground Result`.
+
+- If you disabled split-view, |vsce| displays the Playground results in
+  a new tab to the right of your Playground labeled
+  :guilabel:`Playground Result`.
+
+- If |vsce| returns these results in |json| format, you may edit these
+  results. If the result returns |json|\-formatted content, |vsce|
+  displays it with |json| syntax highlighting and includes an
+  :guilabel:`Edit Document` link at the top of the document.
+
+In the Playground results tab, you can:
```

Comment:
Also above you say that in a split-view mode this is a `pane` and if it is not a split-view mode this is `a new tab`. And here you say `In the Playground results tab, you can`, it makes an impression that in pane you can not `Save the results to a file` etc. Despite the position of the window is completely the same window and has all the same actions.